lib/engine/game/g_1848/game.rb

```diff
@@ -611,7 +623,7 @@ def check_sydney_adelaide_connected
 
         def place_home_token(entity)
           return super if entity.name != :COM
-          return unless @sydney_adelaide_connected
+          return unless @sydney_adelaide_connected || @com_can_operate
```

Comment:
recommend making com_can_operate? a method rather than managing the two different triggers independently throughout the code base.